backend/data/raw/archive/groups_2020_04_27.csv

```diff
@@ -1557,7 +1557,7 @@ Devon,50.7155591,-3.530875,Devon Mutual Aid,https://www.facebook.com/groups/2059
 Devon,50.7155591,-3.530875,Help during covid-19,https://www.facebook.com/groups/201564581177296/,VISIBLE,facebook group,,Devon & Cornwall Police,http://www.devon-cornwall.police.uk,Alphington,http://www.devon-cornwall.police.uk/your-area/teams/Exeter/overview,exeterneighbourhood@devonandcornwall.pnn.police.uk,16/03/2020 06:20:43,eb4b69c3-8dbd-4ccd-a883-28939f8a0aec

 Devonshire Road SE23,51.4444948,-0.051677,Devonshire Road SE23 - Mutual Aid,https://www.facebook.com/devonshire-road-se23-mutual-aid-110826377211337/,VISIBLE,facebook group,,Metropolitan Police Service,http://www.met.police.uk/,Forest Hill,http://www.met.police.uk/a/your-area/met/lewisham/forest-hill/,,16/03/2020 20:19:44,7ab4bde0-fb73-4c05-b911-c07dcabe5310

 Dewsbury,53.689833,-1.6296949,Dewsbury South Covid-19 Community Action Group,https://fb.me/dewsburysouthcovid19,UPDATED,facebook group,,West Yorkshire Police,https://www.westyorkshire.police.uk/my-neighbourhood,Dewsbury,https://www.westyorkshire.police.uk/my-neighbourhood/kirklees/dewsbury,Dewsbury_Mirfield@westyorkshire.pnn.police.uk,22/03/2020 09:37:19,a16e02d3-13f5-4bec-81e9-c5d8a971092a

-"Dewsbury WF13, UK",53.6903786,-1.6467385,20:20 Foundarion Trust ,https://www.facebook.com/2020foundation.org/,TODO,facebook group,,West Yorkshire Police,https://www.westyorkshire.police.uk/my-neighbourhood,Dewsbury,https://www.westyorkshire.police.uk/my-neighbourhood/kirklees/dewsbury,Dewsbury_Mirfield@westyorkshire.pnn.police.uk,3/31/2020 11:25:44,58ffa4a4-4724-4497-aa82-26b30f1c838f

+"Dewsbury WF13, UK",53.6903786,-1.6467385,20:20 Foundarion Trust ,https://www.facebook.com/2020foundation.org/,VISIBLE,facebook group,,West Yorkshire Police,https://www.westyorkshire.police.uk/my-neighbourhood,Dewsbury,https://www.westyorkshire.police.uk/my-neighbourhood/kirklees/dewsbury,Dewsbury_Mirfield@westyorkshire.pnn.police.uk,3/31/2020 11:25:44,58ffa4a4-4724-4497-aa82-26b30f1c838f


```

Comment:
Why the `archive` folder in `raw`?

Reply:
archive function has been disabled a la changelog, will delete this folder